### PR TITLE
Return the modified entity from setting module

### PR DESCRIPTION
**Pull request by mdellweg**
Again, i have been up to a little experiment. Since we now have the new `naildown_entity`, we easily pass the returned information on.

### CODE REVIEW

File: modules/foreman_setting.py [hunk 1]
```diff
@@ -81,14 +81,17 @@
     name: "http_proxy"
 '''
 
-RETURN = ''' # '''
+RETURN = '''
+entity:
```
Comment (sean797) [inline modules/foreman_setting.py]: I'm not sure calling this `entity` is user-friendly, any other suggestions?

Comment (mdellweg) [reply modules/foreman_setting.py]: could be `setting` but maybe `foreman_setting` is better in avoiding confusion.

Comment (sean797) [reply modules/foreman_setting.py]: either works for me :+1:

File: modules/foreman_setting.py [hunk 2]
```diff
@@ -81,14 +81,17 @@
     name: "http_proxy"
 '''
 
-RETURN = ''' # '''
+RETURN = '''
+foreman_setting:
+  description: Created / Updated state of the setting
```
Comment (ehelms) [inline modules/foreman_setting.py]: Is this how the API returns it or a formatted version of the API?

Comment (mdellweg) [reply modules/foreman_setting.py]: Sorry, i do not understand the question.
Right now, this module returns `entity.to_json_dict()`, which is what the nailgun thinks to be the API.

**Comment by akofink** [comment]
@ehelms good to merge?

**Comment by sean797** [comment]
> Side note: this module isn't idempotent. :(

@akofink are you sure? The tests are :green_apple: ?

https://github.com/theforeman/foreman-ansible-modules/blob/1cd5b4d38f70152a4d546cef3bef54060fec9cb7/test/test_playbooks/setting.yml#L15-L26

**Comment by mdellweg** [comment]
What is the status here?
Is your problem related to the changes here? @akofink

**Comment by akofink** [comment]
Not related, I think this is good to merge.

**Pull request merged**